warhorn/default_config.xml

```diff
@@ -37,7 +37,7 @@
 
     Fill this out with the details of the repository that you want to clone
 
-    <tools url="http://repository/one/url.git" label="branch-name" clone="yes">
```

Comment:
After deleting label, the log looks like this
-I- tools cloned successfully.
-I- No tags were found in tools. It would be set to the default (master) branch.
-I- DONE 1
-I- DONE 1
Is this intended behavior?

Reply:
Yes. master is the only branch available in this repo and hence this info gets printed and this was not newly introduced here.

Reply:
oops, What I meant to ask is
is the DONE 1 necessary in this case?
Since we successfully cloned and switched to a branch,
why'd warhorn print DONE 1 in this situation

Reply:
Hi @kaheichan how are you getting DONE 1? For me, it gives DONE 0 as follows:
-I- Git is available
-I- ve_name: war_virtualenv
-I- destination: /data/users/asethura/warrior/warhorn-1120/warriorframework/warhorn/war_virtualenv
New python executable in /data/users/asethura/warrior/warhorn-1120/warriorframework/warhorn/war_virtualenv/bin/python2.7
Not overwriting existing python script /data/users/asethura/warrior/warhorn-1120/warriorframework/warhorn/war_virtualenv/bin/python (you must use /data/users/asethura/warrior/warhorn-1120/warriorframework/warhorn/war_virtualenv/bin/python2.7)
Installing setuptools, pip, wheel...done.
-I- The ability to upgrade/downgrade wariorframework version would be available for use soon.
-I- Verifying if ssh://git@rtx-swtl-git.fnc.net.local:7999/wsan/warhorn_tools.git is a valid git repository.
-I- ssh://git@rtx-swtl-git.fnc.net.local:7999/wsan/warhorn_tools.git is available
-I- Cloning tools.
Cloning into 'warhorn_tools'...
remote: Counting objects: 10, done.
remote: Compressing objects: 100% (6/6), done.
remote: Total 10 (delta 2), reused 0 (delta 0)
Receiving objects: 100% (10/10), done.
Resolving deltas: 100% (2/2), done.
Checking connectivity... done.
-I- tools cloned successfully.
-I- No tags were found in tools. It would be set to the default (master) branch.
-I- DONE 0
-I- DONE 0

Reply:
Found the reason of DONE 1, my virtualenv settings is wrong. confirmed the fix when clone is set to no it doesn't clone, and also test it clone when it is set to yes